plugins/GSdx/GSRendererDX11.cpp

```diff
@@ -37,10 +43,54 @@ bool GSRendererDX11::CreateDevice(GSDevice* dev)
 	return true;
 }
 
-void GSRendererDX11::SetupIA()
+void GSRendererDX11::SetupIA(const float& sx, const float& sy)
 {
 	GSDevice11* dev = (GSDevice11*)m_dev;
 
+	D3D11_PRIMITIVE_TOPOLOGY t;
+
+	bool unscale_hack = UserHacks_unscale_pt_ln  & (GetUpscaleMultiplier() > 1);// &GLLoader::found_geometry_shader;
```

Comment:
Ultimately it would be nice to have a common function for all renderers. But we need to handle `Q division / line to sprite / wild arm hack` in a single function. Besides maybe it would be nicer to directly copy the result in the gl/dx buffer (map/unmap). Rather than to update a local buffer and then copy it in gl/dx buffer.